### ISSUE TITLE
defaultDate option messes up the datepicker

### ISSUE THREAD

**Issue by aalvrz** [DustinFisher/datetimepicker-example#1]
When I try adding a `defaultDate` option to populate the date picker in my edit view, it successfully loads the date. However, it is loaded with incorrect format (defined using `format`), and the date picker stops working at all...

Did you run into this issue?

**Comment by DustinFisher** [comment]
Yeah, I just went in and changed the format and it stays in the YYYY-MM-DD no matter what is put in the format option. I will have to take a closer look at that.
